packages/eslint-config-mc-app/index.js

```diff
@@ -140,9 +127,23 @@ module.exports = {
         'jest/globals': true,
       },
       rules: {
+        ...craRules.jest,
```

Comment:
The fix is that the jest/rtl rule overrides must be specified after the default CRA rules.